dist/radar_client.js

```diff
@@ -381,6 +381,16 @@ Client.prototype._createManager = function() {
   });
 
   manager.on('activate', function() {
+    var socket = client._socket;
+    if (!client.name) {
+      client.name = socket.id;
```

Comment:
it can be a guid independent of the socket id. (It has to be, if it needs to restore subscriptions based on it). On every reconnect, the socket.id changes.